casepro/msgs/models.py

```diff
@@ -417,6 +429,15 @@ def get_history(self):
         """
         return self.actions.select_related('created_by', 'label').order_by('-pk')
 
+    def get_lock(self, user_id):
```

Comment:
I'm pretty sure the user object is always in the context thanks to the `auth` context processor (i.e. `context['user']`). And even if it wasn't, you could just put the user object rather than the id in at https://github.com/rapidpro/casepro/pull/220/files#diff-23f5fc0c123c019ee596bef56b7f1e89R211.

At the model level, it's better to be passing around objects rather than database ids where possible.